docs/parameter_tables/MusMus/DomesticusEurope_1F22.csv

```diff
@@ -0,0 +1,116 @@
+Population size,2040,Pop. size during 55th time interval
+Population size,2040,Pop. size during 56th time interval
+Time (yrs.),1915544,Begining of 1st time interval
+Time (yrs.),1654653,Begining of 2nd time interval
+Time (yrs.),1429281,Begining of 3rd time interval
+Time (yrs.),1234595,Begining of 4th time interval
+Time (yrs.),1066418,Begining of 5th time interval
+Time (yrs.),921140,Begining of 6th time interval
+Time (yrs.),795646,Begining of 7th time interval
+Time (yrs.),687237,Begining of 8th time interval
+Time (yrs.),593589,Begining of 9th time interval
+Time (yrs.),512693,Begining of 10th time interval
+Time (yrs.),442812,Begining of 11th time interval
+Time (yrs.),382446,Begining of 12th time interval
+Time (yrs.),330300,Begining of 13th time interval
+Time (yrs.),285254,Begining of 14th time interval
+Time (yrs.),246340,Begining of 15th time interval
+Time (yrs.),212726,Begining of 16th time interval
+Time (yrs.),183689,Begining of 17th time interval
+Time (yrs.),158606,Begining of 18th time interval
+Time (yrs.),136938,Begining of 19th time interval
+Time (yrs.),118221,Begining of 20th time interval
+Time (yrs.),102052,Begining of 21th time interval
+Time (yrs.),88084,Begining of 22th time interval
+Time (yrs.),76019,Begining of 23th time interval
+Time (yrs.),65596,Begining of 24th time interval
+Time (yrs.),56592,Begining of 25th time interval
+Time (yrs.),48815,Begining of 26th time interval
+Time (yrs.),42096,Begining of 27th time interval
+Time (yrs.),36292,Begining of 28th time interval
+Time (yrs.),31279,Begining of 29th time interval
+Time (yrs.),26948,Begining of 30th time interval
+Time (yrs.),23207,Begining of 31th time interval
+Time (yrs.),19975,Begining of 32th time interval
+Time (yrs.),17183,Begining of 33th time interval
+Time (yrs.),14772,Begining of 34th time interval
+Time (yrs.),12688,Begining of 35th time interval
+Time (yrs.),10889,Begining of 36th time interval
+Time (yrs.),9334,Begining of 37th time interval
+Time (yrs.),7991,Begining of 38th time interval
+Time (yrs.),6831,Begining of 39th time interval
+Time (yrs.),5829,Begining of 40th time interval
+Time (yrs.),4964,Begining of 41th time interval
+Time (yrs.),4216,Begining of 42th time interval
+Time (yrs.),3570,Begining of 43th time interval
+Time (yrs.),3012,Begining of 44th time interval
+Time (yrs.),2530,Begining of 45th time interval
+Time (yrs.),2114,Begining of 46th time interval
+Time (yrs.),1754,Begining of 47th time interval
+Time (yrs.),1443,Begining of 48th time interval
+Time (yrs.),1175,Begining of 49th time interval
+Time (yrs.),943,Begining of 50th time interval
+Time (yrs.),743,Begining of 51th time interval
+Time (yrs.),570,Begining of 52th time interval
+Time (yrs.),420,Begining of 53th time interval
+Time (yrs.),291,Begining of 54th time interval
+Time (yrs.),180,Begining of 55th time interval
+Time (yrs.),83,Begining of 56th time interval
+Time (yrs.),0,Begining of 57th time interval
```

Comment:
Parameter table looks okay. I think that the standard is to have one more entry in the list of Nes than in the list of times. This should correspond to the Ne before the first time interval (the ancestral Ne). Other Nes should map to time intervals. This also relates to my comment on the "shift" in the demographic model